tests/test_registerutils.py

```diff
@@ -323,6 +326,7 @@ def test_clean_host_file_no_empty_bottom_lines():
 
 # Added by SMT, please, do NOT remove this line
 2.3.4.5   smt-entry.susecloud.net smt-entry
+2.3.4.5   registry-entry.susecloud.net
```

Comment:
We should create a test case where the user interfered in the order see the example in a previous comment